### PR TITLE
Update types

### PR DESCRIPTION
**Pull request by oTkPoBeHuE**

### CODE REVIEW

File: src/types/UrbanBotInstance.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { UrbanNativeEvent, UrbanSyntheticEvent } from './Events';
+import { UrbanMessage, UrbanExistingMessage } from './Messages';
+
+export type ProcessUpdate<NativeEvent extends UrbanNativeEvent = UrbanNativeEvent> = (
+    event: UrbanSyntheticEvent<NativeEvent>,
+) => void;
+
+export interface UrbanBot<NativeEvent extends UrbanNativeEvent = UrbanNativeEvent, MessageMeta = any> {
```
Comment (ledamint) [inline src/types/UrbanBotInstance.ts]: unknown?

Comment (ledamint) [inline src/types/UrbanBotInstance.ts]: UrbanBotType? And keep UrbanBot name as an interface

File: src/components/Root.tsx [hunk 2]
```diff
@@ -41,41 +41,42 @@ function Chat<NativeEvent extends UrbanNativeEvent, MessageMeta>({
...
+export type RootProps<Bot extends UrbanBot> = {
+    bot: UrbanBotInstance<Bot>;
     children: React.ReactNode;
     timeToClearUserSession?: number;
     isNewMessageEveryRender?: boolean;
     parseMode?: UrbanParseMode;
 };
 
-export function Root<NativeEvent extends UrbanNativeEvent = UrbanNativeEvent, MessageMeta = unknown>({
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (ledamint) [inline src/components/Root.tsx:52]: Is it possible to fix it?

Comment (oTkPoBeHuE) [reply src/components/Root.tsx:52]: no

File: src/types/UrbanBotInstance.ts (hunk 1 above)
Comment (oTkPoBeHuE) [reply src/types/UrbanBotInstance.ts]: fix

**Pull request merged**